_libraries/Lodata.md

```diff
@@ -0,0 +1,14 @@
+---
+category: other
+name: Lodata
+link: https://github.com/flat3/lodata
+version: V4 
+object: Server
+downloads:
+  - source: GitHub
+    link: https://github.com/flat3/lodata
+  - source: Packagist
+    link: https://packagist.org/packages/flat3/lodata
+---
+Lodata is a fully-featured OData 4.01 producer for the Laravel web application framework.
```

Comment:
```suggestion
Lodata is a fully-featured OData v4.01 producer for the Laravel web application framework.
```

Reply:
@27pchrisl could you add this suggestion and so I can publish Asap.